ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -118,22 +133,88 @@ public void setArguments(Map<String, Object> arguments) {
     }
   }
 
+  public class CqlTransformRequest {
+    private List<CqlRequestImpl> searches;
+    private int count;
+    private List<CqlRequest.Sort> sorts;
+    private List<String> hiddenResults;
+
+    public CqlTransformRequest() {
+      this.searches = Collections.emptyList();
+      this.count = 0;
+      this.sorts = Collections.emptyList();
+      this.hiddenResults = Collections.emptyList();
+    }
+
+    public void setSearches(List<CqlRequestImpl> searches) {
+      this.searches = searches;
+    }
+
+    public List<CqlRequestImpl> getSearches() {
+      return this.searches;
+    }
+
+    public void setCount(int count) {
+      this.count = count;
+    }
+
+    public int getCount() {
+      return this.count;
+    }
+
+    public void setSorts(List<CqlRequest.Sort> sorts) {
+      this.sorts = sorts;
+    }
+
+    public List<CqlRequest.Sort> getSorts() {
+      return this.sorts;
+    }
+
+    public List<CqlRequestImpl> getCqlRequests() {
```

Comment:
✏️ This method seems out of place. What do you think about doing this transformation inline and removing this method?